### PR TITLE
Initiate API spec

### PR DESCRIPTION
**Pull request by debovema**
A simple Swagger API spec to review and enhance

### CODE REVIEW

**Review by ffassler — CHANGES_REQUESTED** [review]
The Smiley definition is not used. Should be removed?

**Review by emilienthomas — CHANGES_REQUESTED** [review]
Would rather use a string for smileyId

**Review by debovema — COMMENTED** [review]
OK to change *smileyId* to a string. This will allow to request the display of a smiley directly with its string representation if I understand well, such as *:some_smiley:* or maybe *:)*.

File: swagger.yaml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      description: "Triggers the display of the selected smiley on the Raspberry screen"
+      operationId: "displaySmileyById"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "smileyId"
+        in: "path"
+        description: "ID of smiley to display"
+        required: true
+        type: "integer"
```
Comment (ffassler) [inline swagger.yaml]:
```suggestion
        type: "string"
```

File: swagger.yaml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        format: "int64"
```
Comment (ffassler) [inline swagger.yaml]:
```suggestion
```

File: swagger.yaml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+      - "smiley"
+      summary: "Display a smiley by ID"
+      description: "Triggers the display of the selected smiley on the Raspberry screen"
+      operationId: "displaySmileyById"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "smileyId"
+        in: "path"
+        description: "ID of smiley to display"
```
Comment (ffassler) [inline swagger.yaml]:
```suggestion
        description: "ID of smiley to display or string representation of smiley like :some_smiley:"
```

**Review by ffassler — CHANGES_REQUESTED** [review]
change smileyId to a string. This will allow to request the display of a smiley directly with its string representation such as :some_smiley:

**Pull request merged**